packages/@aws-cdk/aws-sagemaker/README.md

```diff
@@ -195,3 +195,60 @@ const endpointConfig = new sagemaker.EndpointConfig(this, 'EndpointConfig', {
   ]
 });
 ```
+
+### Endpoint
+
+When you create an endpoint from an `EndpointConfig`, Amazon SageMaker launches the ML compute
+instances and deploys the model or models as specified in the configuration. To get inferences from
+the model, client applications send requests to the Amazon SageMaker Runtime HTTPS endpoint. For
+more information about the API, see the
+[InvokeEndpoint](https://docs.aws.amazon.com/sagemaker/latest/dg/API_runtime_InvokeEndpoint.html)
+API. Defining an endpoint requires at minimum the associated endpoint configuration:
+
+```typescript
+import * as sagemaker from '@aws-cdk/aws-sagemaker';
+
+declare const endpointConfig: sagemaker.EndpointConfig;
+
+const endpoint = new sagemaker.Endpoint(this, 'Endpoint', { endpointConfig });
+```
+
+### AutoScaling
+
+To enable autoscaling on the production variant, use the `autoScaleInstanceCount` method:
+
+```typescript
+import * as sagemaker from '@aws-cdk/aws-sagemaker';
+
+declare const endpointConfig: sagemaker.EndpointConfig;
+
+const endpoint = new sagemaker.Endpoint(this, 'Endpoint', { endpointConfig });
+const productionVariant = endpoint.findInstanceProductionVariant('variantName');
```

Comment:
how would you know the variant name here? Is it defined as part of `endpointConfig`? If so, I think it's worth defining the endpointConfig completely and pulling `variantName` out into a variable, at least for this example so it's a bit more straightforward.

Reply:
> how would you know the variant name here? Is it defined as part of `endpointConfig`? If so, I think it's worth defining the endpointConfig completely and pulling `variantName` out into a variable, at least for this example so it's a bit more straightforward.

Correct, and that feedback sounds good. Should I follow the same pattern in the "_Metrics_" section since it too performs `findInstanceProductionVariant`?

Reply:
I'm ok with it just being here. This example focuses on a method exposed on the `productionVariant`, so it makes sense to be a little more explicit about how we're getting the variant.